docs/breaking-changes/v2.md

```diff
@@ -0,0 +1,82 @@
+# Breaking Changes for v2
```

Comment:
Should this be v3 instead? There is already a published v2 that is pointing to `/static-analysis/semgrep` that might be in use and will start failing

Reply:
Yes. I'll change it to v3. Do we want to add it as doc/breaking-changes/v3.md or under /static-analysis/v3.md?